envbuilder.go

```diff
@@ -656,7 +656,11 @@ func Run(ctx context.Context, options Options) error {
 				Insecure:      options.Insecure,
 				InsecurePull:  options.Insecure,
 				SkipTLSVerify: options.Insecure,
-				RegistryMirrors: strings.Split(os.Getenv("KANIKO_REGISTRY_MIRROR"), ";"),
+				// Enables registry mirror features in Kaniko, see more in link below
+				// https://github.com/GoogleContainerTools/kaniko?tab=readme-ov-file#flag---registry-mirror
+				// Related to PR #114
+				// https://github.com/coder/envbuilder/pull/114
+				RegistryMirrors: strings.Split(os.Getenv("·"), ";"),
```

Comment:
I think changing the env var here was a typo!

Reply:
If the environment variable is unset, this will return `[""]`. Will that give the correct behavior? I would have thought we want to leave this `nil` when there are no mirrors.

Reply:
Thank you for pointing out that I forgot to well consider the case when the environment variable is not set.
It's now fixed. :)